helpers/prometheus.py

```diff
@@ -378,3 +379,21 @@ def collect(self):
                     )
 
             yield project_events_metrics
+
+        if self.rate_limit_metrics == "True":
+            project_rate_metrics = CounterMetricFamily(
+                "sentry_rate_limit_second",
+                "Rate limit per project",
```

Comment:
Since the rate limit refers to the number of events per second that DSN can send, what about add this to the metric name?
```suggestion
                "sentry_rate_limit_events_sec",
                "Rate limit events per second for a project",
```